src/middlewared/middlewared/plugins/device_/vrrp_events.py

```diff
@@ -37,13 +38,22 @@ def unpause_events(self):
         if VrrpObjs.event_thread is not None and VrrpObjs.event_thread.is_alive():
             VrrpObjs.event_thread.unpause()
 
+    def set_non_crit_ifaces(self):
+        if VrrpObjs.event_thread is not None and VrrpObjs.event_thread.is_alive():
```

Comment:
We should remove is_alive check as currently we set the value on a thread that is not yet started